lib/internal/streams/writable.js

```diff
@@ -94,6 +96,12 @@ const kBufferProcessing = 1 << 16;
 const kPrefinished = 1 << 17;
 const kAllBuffers = 1 << 18;
 const kAllNoop = 1 << 19;
+const kHasOnFinished = 1 << 20;
+const kHasErrored = 1 << 21;
+
+const kCorkedShift = 22;
+const kCorkedMask = 0b1111
+const kCorked = kCorkedMask << kCorkedShift; // 4 bits
```

Comment:
Add a comment about this? Not sure I would have been able to figure this out easily had we not synced about these ideas

Reply:
Suggestion?